Challenges/madlibs_pt2.rb

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+
+
+puts "--" *10
+puts "|Madlibs Blanks|"
+puts "--" *10
+
+blanks = ['adjective', 'plural Noun', 'noun', 'adjective', 'part of the body', 'adjective',' plural noun', 'part of the body', 'adjective', 'adverb']
+
+vowels = ['a', 'e'. 'i', 'o', 'u']
+
+answers = blanks.map do |request|
+    article = vowels.include?(request[0]? 'an : a')
```

Comment:
You have a typo in the ternary operator. Also, what are you trying to do with this line of code? I don't think it's doing quite what you'd expect.